android/src/main/kotlin/org/torusresearch/flutter/customauth/CustomAuthPlugin.kt

```diff
@@ -75,92 +74,116 @@ class TorusDirectPlugin : FlutterPlugin, ActivityAware, MethodCallHandler {
+                    "publicAddress=${torusResponse.publicAddress}"
                 )
                 return mapOf(
-                        "publicAddress" to torusResponse.publicAddress,
-                        "privateKey" to torusResponse.privateKey
+                    "publicAddress" to torusResponse.publicAddress,
+                    "privateKey" to torusResponse.privateKey,
+                    "userInfo" to mapOf(
+                        "email" to torusResponse.userInfo.email,
+                        "name" to torusResponse.userInfo.name,
+                        "profileImage" to torusResponse.userInfo.profileImage,
+                        "verifier" to torusResponse.userInfo.verifier,
+                        "verifierId" to torusResponse.userInfo.verifierId,
+                        "typeOfLogin" to torusResponse.userInfo.typeOfLogin,
+                        "accessToken" to torusResponse.userInfo.accessToken,
+                        "idToken" to torusResponse.userInfo.idToken
+                    )
                 )
             }
             "triggerAggregateLogin" -> {
                 val torusDirectSdk =
-                        TorusDirectSdk(torusDirectArgs, activity ?: context)
+                    CustomAuth(torusDirectArgs, activity ?: context)
                 val torusResponse = torusDirectSdk.triggerAggregateLogin(
-                        AggregateLoginParams(
-                                AggregateVerifierType.valueOfLabel(call.argument<String>("aggregateVerifierType")),
-                                call.argument<String>("verifierIdentifier"),
-                                (call.argument<List<Map<String, Any>?>>("subVerifierDetailsArray")!!.map { mapSubVerifierDetails(it) }).toTypedArray()
-                        )
+                    AggregateLoginParams(
+                        AggregateVerifierType.valueOfLabel(call.argument<String>("aggregateVerifierType")),
+                        call.argument<String>("verifierIdentifier"),
+                        (call.argument<List<Map<String, Any>?>>("subVerifierDetailsArray")!!
+                            .map { mapSubVerifierDetails(it) }).toTypedArray()
+                    )
                 ).join()
                 Log.d(
-                        "${TorusDirectPlugin::class.qualifiedName}#triggerAggregateLogin",
-                        "publicAddress=${torusResponse.publicAddress}"
+                    "${CustomAuthPlugin::class.qualifiedName}#triggerAggregateLogin",
+                    "publicAddress=${torusResponse.publicAddress}"
                 )
                 return mapOf(
-                        "publicAddress" to torusResponse.publicAddress,
-                        "privateKey" to torusResponse.privateKey
+                    "publicAddress" to torusResponse.publicAddress,
+                    "privateKey" to torusResponse.privateKey,
+                    "userInfo" to listOf(
```

Comment:
since you added this, then you need to add this params to TorusCredentials class in lib/torus_direct.dart
This should be nullable fields.

Reply:
please see if the latest version fits